### ISSUE TITLE
rbf for mac

### ISSUE THREAD

**Issue by thecrimplord** [ethan-ou/camera-match#3]
Hey Ethan,
any news on getting RBF to work on mac? I tried installing that version but had no luck.
Thanks

**Comment by ethan-ou** [comment]
I don't think so because it requires the underlying library to support Mac. I'm unable to rewrite the library to make it work purely on Python either, so I don't think this possible for now.

**Comment by chromanista** [comment]
Bumping this thread - @ethan-ou I think `scipy.interpolate.
RBFInterpolator` could work as a replacement for `xalglib`. It looks like it may have been too new at the time the original RBF LUT code was written.

I'm currently juggling a few things, but might be able to share a proof-of-concept in a bit.

**Comment by ethan-ou** [comment]
The scipy version only supports 2D interpolation as far as I can see and a LUT requires 3D. Also the library used seems to also have some regularisation because when comparing to other RBF implementations I found this gave the best results. Not all RBF's will give the same output if they contain extra tricks like with xalglib.

**Comment by ethan-ou** [comment]
In the latest version of camera-match I've deprecated RBF since it's slow and doesn't actually give good quality matches! I recommend using LUT2D and LUT3D which uses standard Python libraries meaning it'll work across platforms.
